libraries/standard/json/cbmc/proofs/skipAnyLiteral/README.md

```diff
@@ -0,0 +1,10 @@
+skipAnyLiteral proof
+==============
+
+This directory contains a memory safety proof for skipAnyLiteral.
```

Comment:
Runtime? Coverage? Assumptions made?

Reply:
trivial runtime, full coverage, no assumptions (that I can think of)

I can add such verbiage to each readme, but it seems to me the report says it better.